palantirnet/federated-search-demo
# D7 terms

## Description

Updates the install of Drupal 7.

## Testing instructions

1. Checkout this branch
1. `vagrant ssh`
1. `phing init`
1. `phing install-d7`

1. Confirm that `web/d7/docroot/sites/all/modules/contrib/search_api_federated_solr` is a symlink to `src/search_api_federated_solr-7.x`
1. Confirm that `search_api_federated_solr-7.x` is on branch `7.x-2.x`
1. Confirm that the site installs with content (including pictures).
